### DISCUSSION TITLE
new theme

### DISCUSSION THREAD

**Discussion by rishav12s** [RAprogramm/neovim-IDE#3] [Ideas]
great project man i have been planning to use neovim with hyprdots and your's is just the way to go just one request could you add the synth wave extra theme for hyprdots to match with it

**Comment by RAprogramm** [comment]
> great project man i have been planning to use neovim with hyprdots and your's is just the way to go just one request could you add the synth wave extra theme for hyprdots to match with it 

Thank you!  Of course I will do this, where can I look at the topic itself?  could you share the link?

**Reply by rishav12s** [reply]
this one https://github.com/prasanthrangan/hyprdots-mod

**Reply by RAprogramm** [reply]
> this one https://github.com/prasanthrangan/hyprdots-mod 

Thank you.  I will do it as soon as possible!

P.S.: do not forget about star)

**Comment by RAprogramm** [comment]
Theme "Synth-Wave" was added. Thank you for idea!
